pkg/ioutils/copy.go

```diff
@@ -0,0 +1,21 @@
+package ioutils
```

Comment:
@LK4D4 Would you like to put this code in `/pkg/pools` instead? It'd be nice to have it all in one place.